src/plcc/load_spec/parse_spec/parse_lexical_spec.py

```diff
@@ -0,0 +1,75 @@
+from dataclasses import dataclass
+import re
+from re import Match
+
+from ..load_rough_spec.parse_lines import Line
+
+@dataclass
+class LexicalRule:
+    line: Line
+    isSkip: bool
+    name: str
+    pattern: str
+
+@dataclass
+class LexicalSpec:
+    ruleList: [LexicalRule|Line]
+
+def parse_lexical_spec(lines: list[Line]) -> LexicalSpec:
+    return LexicalParser(lines).parseLexicalSpec()
+
+class LexicalParser():
+    def __init__(self, lines: [Line]):
+        self.lines = lines
+        self.patterns = {
+            'skipToken' : re.compile(r'^skip\s+(?P<Name>\S+)\s+(?P<Pattern>((\'\S+\')|(\"\S+\")))\s*(?:#.*)*$'),
+            'tokenToken' : re.compile(r'(?:^token\s+)?(?P<Name>\S+)\s+(?P<Pattern>((\'\S+\')|(\"\S+\")))\s*(?:#.*)*$')
+        }
+        self.spec = LexicalSpec([])
+
+    def parseLexicalSpec(self) -> LexicalSpec:
+        if not self.lines:
+            return self.spec
+        for line in self.lines:
+            if self._isBlankOrComment(line.string.strip()):
+                continue
+            lineIsSkipToken, lineIsRegularToken = self._matchToken(line.string)
+            if lineIsSkipToken or lineIsRegularToken:
+                self.spec.ruleList.append(self._generateTokenRule(line, lineIsSkipToken, lineIsRegularToken))
+            else:
+                self.spec.ruleList.append(line)
```

Comment:
Everything looks much better, however this section can be split into a helper method, and then I think you should be all good, as it reduces the length of the current method, you can also get rid of the continue if you choose to:
```
for line in self.lines:
    if not self._isBlankOrComment(line):
        self._processLine(line) # you can name your helper function whatever you choose
```